app.js

```diff
@@ -0,0 +1,185 @@
+var timeLabelsArray = ['10AM: ','11AM: ','12PM: ', '1PM: ', '2PM: ', '3PM: ', '4PM: ', '5PM: ', 'SUM: ']
+var pikePlace = {
+  min: 17,
+  max: 88,
+  avg: 5.2,
+  randomOutput: function(){
+    return Math.floor(((Math.random()*(this.max - this.min + 1)) + this.min) * this.avg);
+  },
+  dailyRandomOutput: function(){
+    var cookiesArray = [];
+    for(i = 0; i < 8; i++){
+      var arrayValue = this.randomOutput();
+      cookiesArray.push(arrayValue);
+    }
+    return cookiesArray;
```

Comment:
Nice work creating your cookiesArray within this function's scope and returning it from the call. Great memory management practice.